js/index.js

```diff
@@ -40,3 +40,43 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+const nav1 = document.getElementsByTagName("a")[1].innerHTML=`${siteContent.nav["nav-item-1"]}`;
```

Comment:
Very good demonstration of using an opening bracket to specify the value you want.

I'd also suggest trying out `querySelectorAll`. Then using a for-loop in a function to automate the changes you want.